common-content/en/energisers/introduce-yourself/index.md

```diff
@@ -0,0 +1,35 @@
++++ 
+title=" Introduce Yourself!" 
+emoji="🎙️"
+time=20 
+[objectives] 
+1="Craft an elevator pitch" 2="Learn about your peers" 
+[build] 
+    render = 'never' 
+    list = 'local' 
+    publishResources = false 
++++
+
+### The task
+
+In your professional life, you will often be asked to introduce yourself. It is a good idea to have an [elevator pitch](https://careerdevelopment.princeton.edu/sites/g/files/toruqf1041/files/media/elevator_pitch.pdf) ready, a brief (30 seconds) way of introducing yourself, getting across a key point or two, and making a connection with someone.
+
+Today we will be going around the room and introducing ourselves!     `
+
+Facilitator(s) start by sharing your elevator pitch and then popcorn around the room.
+
+Some tips:
+- Your response should be around 30 seconds.
+- Have a list of items ready but tailor what you say to your audience. For example, at a conference don't tell them you love making cupcakes but on the first day of a new job introducing yourself to your new team feel free to add 1-2 personal details.
```

Comment:
```suggestion
- Have a list of items ready but tailor what you say to your audience. For example, at a technical conference focus on your technical interests rather than your love of making cupcakes, but on the first day of a new job introducing yourself to your new team feel free to add 1-2 personal details.
```